src/components/BookDetailPage/BookReviewTab.tsx

```diff
@@ -0,0 +1,301 @@
+import { useState } from 'react';
+import {
+  Box,
+  Typography,
+  Button,
+  Grid,
+  Card,
+  CardContent,
+  Avatar,
+  Stack,
+} from '@mui/material';
```

Comment:
<img width="616" alt="image" src="https://github.com/user-attachments/assets/ac5cd220-13d5-4e69-be69-aae2d86e81ab" />

그냥 [Grid 컴포넌트](https://mui.com/material-ui/react-grid/)는 deprecated 되었다고 해서 [Grid v2](https://mui.com/material-ui/react-grid2/) 별도로 import해서 사용해야 할 것 같아요!

```suggestion
import {
  Box,
  Typography,
  Button,
  Card,
  CardContent,
  Avatar,
  Stack,
} from '@mui/material';
import Grid from '@mui/material/Grid2';
```

Reply:
Grid2 반영해서 코드도 수정이 필요할 것 같긴하네요! import문만 바꾸면 UI가 깨지는 군요..!

Reply:
수정하겠습니다!